bundle/manifests/vertical-pod-autoscaler.clusterserviceversion.yaml

```diff
@@ -702,6 +703,9 @@ spec:
                   initialDelaySeconds: 5
                   periodSeconds: 10
                 resources:
+                  limits:
+                    cpu: 100m
+                    memory: 100Mi
```

Comment:
Are we sure this is high enough that nobody will hit this? Like if they're set to auto-upgrade it and they're over the limit, they'd pull it and die to OOM-kill, right?

Reply:
Yeah, I can just remove it especially since we didn't cap it before, and I think I have to remove the limits on the rbac-proxy container too. That might happen if our operator is behaving badly for some reason and is over the limit.

Reply:
Yeah, I wasn't implying it would sneak the limit in -- it would restart, but I just want to make sure we know we're not close. 

For history, Back when I was on the MCO, OpenShift turned on the FIPS compliance shim/builder thing, and the additional stuff it did to link the FIPS libraries put us over our memory limit. Customer MCOs started dying of OOM after an upgrade, and we were very confused for a little while because we didn't change anything, and we weren't leaking, but we were all of a sudden over. I think it was [OCPBUGS-17568](https://issues.redhat.com/browse/OCPBUGS-17568)

( MCO also had the problem though where our object size varied a lot, and people could feed us really fat manifests, whereas I don't think anyone is feeding us really fat VPA controllers ) 

Anyway, I just wanted to make sure there is enough slack so it's only protecting the customer's environment if we go haywire, and that we're not so close to the edge that if the libraries get fatter we start dying sometimes and upsetting them.  

I'd probably feel better about a 200+ mem limit, 500 would be just fine.  :smile:

Reply:
I agree here. Let's change it to 500 MiB which gives a lot of head-room, but protects clusters against unforeseen memory leak bugs bringing a node to its knees.